src/schema/schema_alter.c

```diff
@@ -91,6 +111,172 @@ __alter_object(WT_SESSION_IMPL *session, const char *uri, const char *newcfg[])
     return (__alter_apply(session, uri, newcfg, WT_CONFIG_BASE(session, object_meta)));
 }
 
+/*
+ * __alter_get_object_id_range --
+ *     Get current and oldest object IDs for the tiered object.
+ */
+static int
+__alter_get_object_id_range(WT_SESSION_IMPL *session, WT_TIERED *tiered, const char *uri,
+  uint32_t *pcurrent_id, uint32_t *poldest_id)
```

Comment:
Style nit: WT generally puts the 'p' at the end of the name so I suggest `current_idp` and `oldest_idp`.